frontend/src/containers/Collection/CollectionDetail/CollectionDetail.js

```diff
@@ -162,16 +152,27 @@ class CollectionDetail extends Component {
         ));
 
         let inviteModal = null;
-        if (this.props.me && this.props.members.map((x) => x.id).includes(this.props.me.id)) {
+        if (this.props.selectedCollection.owned
+            || (this.props.me && this.props.members.map((x) => x.id).includes(this.props.me.id))) {
```

Comment:
This code still has a potential problem. As you know, `/api/user/collection` only gives 10 users per call. Therefore, if a user is one of members(not owner) and the first response of `/api/user/collection`, this code will wrongly think the user is not a member of the collection. Maybe a key showing whether `request_user` is a member of the collection will be needed, as `owned` was.